exampleSite/content/posts/theme-documentation-basics.md

```diff
@@ -42,266 +41,316 @@ git submodule add https://github.com/526avijitgupta/gokarna.git themes/gokarna
 
-This is the image url for the avatar on the homepage and the header.
+### Accent color
+
+The color displayed when a user hovers over hyperlinks (`<a>` tags), expressed as a [hexadecimal](https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color).
 
 ```toml
 [params]
-  avatarURL = "/images/avatar.jpg"
+  accentColor = "#FF4D4D"
 ```
 
-### Avatar Size
-
-You have an option to change the avatar size on the homepage. Options are: `size-xs`, `size-s`, `size-m`, `size-l` & `size-xl`. (Default: `size-m`)
+### Avatar URL
 
-```toml
-[params]
-  avatarSize = "size-l"
-```
+The avatar's resource URL, displayed on the [home page](/) and header (top-left).
 
-### Description
+Images are typically placed into the [`assets/`](https://gohugo.io/getting-started/directory-structure/#assets) or [`static/`](https://gohugo.io/getting-started/directory-structure/#static) directories (which are copied to the top-level directory [at build time](#build-the-website)).
 
-Description to display on homepage below the title and avatar.
 ```toml
 [params]
-  description = "Hello, world!"
+  avatarURL = "/images/avatar.webp"
 ```
 
-### Accent Color
+### Avatar size
 
-Accent color is displayed when you hover over `<a>` tags. It takes a hex/rgb color code. Default is `#FF4D4D`
+Set the avatar's size as: `size-xs`, `size-s`, `size-m`, `size-l`, or `size-xl`.
 
 ```toml
 [params]
-  accentColor = "#08F"
+  avatarSize = "size-m"
 ```
 
-### Posts on home page
-
-If you want to display posts on the homepage, the options are:
-
-- `popular`: Show popular posts on home page if the value is set to popular. It sorts the all the posts by it’s weight attribute in ascending order. Read more about it [here](/posts/theme-documentation-advanced/#weights).
-- `recent`: Show recent posts on home page if the value is set to recent
-- Do not show anything if the variable is unset or an empty string.
+### Custom Head HTML
 
-You can define how many posts will be displayed on homepage by setting `numberPostsOnHomePage`. If `numberPostsOnHomePage` is absent (empty) or zero the default value is used.
+Add arbitrary HTML code to the [`<head>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head).
 
 ```toml
 [params]
-  showPostsOnHomePage = "popular"
-  numberPostsOnHomePage = 3
+  customHeadHTML = """
+    <!-- KaTeX -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.0/dist/katex.min.css" integrity="sha384-Xi8rHCmBmhbuyyhbI88391ZKP2dmfnOl4rT9ZfRI7mLTdk1wblIUnrIq35nqwEvC" crossorigin="anonymous">
+    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.0/dist/katex.min.js" integrity="sha384-X/XCfMm41VSsqRNQgDerQczD69XqmjOOOwYQvr/uuC+j4OPoNhVgjdGFwhvN02Ja" crossorigin="anonymous"></script>
+    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.0/dist/contrib/auto-render.min.js" integrity="sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR" crossorigin="anonymous"></script>
```

Comment:
The syntax highlighting gets messed up on rendering 🥲 

<img width="633" alt="image" src="https://github.com/user-attachments/assets/173fc56b-a70e-4ca1-90be-da9e6080ed0d">

Reply:
I've reverted this commit to the previous version: the exampleSite lines were too long to render without scrollbars (even when broken), and using HTML syntax highlighting (instead of TOML) produced an arguably uglier result